src/entity/User.ts

```diff
@@ -94,6 +94,10 @@ export class User {
   @Index('IDX_user_referral')
   referralId?: string | null;
 
+  @Column({ length: 36, nullable: true })
```

Comment:
@idoshamun I copied length from referral, I think it is plenty but we might wanna double to be save if we have lengthier origins?

Reply:
Also, do we have referrals outside of squads, asking because we might be able to prefill `squad` to existing users with `referralId`?

Reply:
Let's use `type: 'text'` instead of setting the length. We can set default value to `squad` as you suggested to prefill evertyhing

Reply:
@idoshamun would we use custom `bin` to pre-fill current values because I don't know if we can conditionally set values of `referralOrigin` depending if `referralId` has value?

Reply:
@capJavert you can add it as statement to the migration to deal with that. Way easier.
Let's also add a check when creating a new user that if referralId is set we will set referralOrigin to squad to have backwards compatibility for users with old version

Reply:
Added in a68d66d6adfacc2516057140f5836f047c469027 and 2932aa49de456303ecf2c075c099379bea8e85fb